### ISSUE TITLE
MIDI file output

### ISSUE THREAD

**Issue by psi29a** [Mindwerks/wildmidi#62]
Since we've crossed the line from exclusive MIDI support to XMI (and possibly others), it would be nice to export the resulting MIDI. This would get rid of the those "if 0" bits of code and help people out that just want to dump their weird/old file formats to standard MIDI.

This would need to be handled in wildplayer with a command line option, like -m FILENAME.

And for completness, provide an interface option in wildmidi_lib.* to allow for just retrieving the new MIDI data.

**Comment by chrisisonwildcode** [comment]
The internal event2midi function is working within -update ... to access it you press 'm' in the player while playing what ever and it will save a type 0 midi file of the current song playing in the current directory using the name of the file currently being played but with a .mid extension.

CAUTION: Will overwrite any existing .mid with the same name in the current directory.

Marking this as good enough for now.

**Comment by psi29a** [comment]
Awesome. :)

**Comment by psi29a** [comment]
Is there also a flag at runtime to dump to midi directly without playback?

If so, please close. :)

**Comment by chrisisonwildcode** [comment]
There is a command line which coverts to midi without playback, these use the converters you guys made which convert directly to MIDI if I remember correctly. 

But all conversion for playback is handled by the converters I wrote which convert the files to the internal event struct, then a single function converts the event struct to type 0 midi on request.

It was done this way to reduce double handling when you only wanted to only convert or play.

Closing this up :)
